public/scripts/projects.js

```diff
@@ -1,30 +1,57 @@
+// 1. break out the js files to CMV files.
+// 2. refractor code to make it more eefficient and lean
+
 
 'use strict';
 var projects = [];
 
+//Models
 function Project(rawDataObject) {
   for (var key in rawDataObject) {
     this[key] = rawDataObject[key];
   }
 }
+
+//refractored - broken
+// Project.prototype.toHtml = () => {
+//   let template = $('#project-template').html();
+//   let templateRender = Handlebars.compile(template);
+//   return templateRender(this);
```

Comment:
probably breaks because you are using this but the lexical arrow function changes the scope of what you are doing. I would avoid lexical arrows when doing object prototypes.